.travis.yml

```diff
@@ -3,8 +3,9 @@ dist: bionic
 language: php
 
 php:
-  - 7.4
-  - 8.0
+  - '7.4'
+  - '8.0'
```

Comment:
```suggestion
  - '8.0'
  - '8.1.0'
```

Reply:
@MPParsley I would prefer `nightly` because >8.1.3 have security vulnerabilities (PHP 8.1.3 was a security release). So, either `>=8.1.3` or `nightly`. I would say let's keep `nightly` until PHP >=8.1.3 is available